src/components/Tooltip/Tooltip.tsx

```diff
@@ -47,6 +48,7 @@ export type TooltipProps = PropsWithChildren<{
     enterDelay?: number;
     open?: boolean;
     disabled?: boolean;
+    /** @deprecated use disabled */
```

Comment:
Maybe we could have a more detailed explanation here? `use disabled since the tooltip is always present in the DOM now so hidden has no effect anymore` or something like that... Not necessary tho. WDYT?

Reply:
Good call, Updated it!